.github/workflows/main.yml

```diff
@@ -51,8 +61,13 @@ jobs:
         uses: actions/checkout@v3
       - name: Free Disk Space
         uses: ./.github/workflows/free-disk-space
+      - name: Mock EIO binary stubs
+        run: |
+          touch ./recipes-app/iot2050-eio-manager/files/bin/iot2050-eiofsd
+          touch ./recipes-app/iot2050-eio-manager/files/bin/map3-fw.bin
+          touch ./recipes-app/iot2050-eio-manager/files/bin/firmware-version
       - name: Build image
-        run: ./kas-container build kas-iot2050-swupdate.yml:kas/opt/secure-boot.yml
+        run: ./kas-container build kas-iot2050-swupdate.yml:kas/opt/secure-boot.yml:kas/opt/eio.yml
```

Comment:
I think we don't need to enable EIO in all three cases because it is not directly affected by the variants. Maybe just append it to the secure boot variant?

Actually, but that is an optimization on top, I could imagine folding the RT option into the secure boot target as well to save resources.

Reply:
Agree and changed.